docs/how-to/debugging/disable-a-linter.rst

```diff
@@ -15,9 +15,10 @@ For example, to disable both built-in linters, add this to your project file:
       ignore:
         - classic
         - library
+        - metadata
 
 
-.. _how-to-disable-a-linter-ignore-specific-files:
+.. _how-to-disable-a-linter-ignore-specific-files-and-field:
```

Comment:
```suggestion
.. _how-to-disable-a-linter-ignore-specific-files:
```

Instead of altering the reference name, let's add a new one after it, and name it after the modified heading. Both refs will then work.